### DISCUSSION TITLE
Function Keys not working in Teleport Connect ssh session

### DISCUSSION THREAD

**Discussion by muhlhorn** [gravitational/teleport#23379] [Q&A]
We have a custom application running on ubuntu that has a command line interface. When running this application through Putty we utilize function keys to navigate. From Teleport Connect some of these function keys do not work. We suspect an encoding issue but cannot pin down the actual cause. Anyone had this issue? Found a solution?

**Comment by webvictim** [comment]
I suspect that as you say, there's some kind of encoding problem.

Can you run `echo $TERM` inside both PuTTY and Teleport Connect after establishing the remote connection and share the outputs?

It's also technically possible to connect to Teleport servers via PuTTY but we haven't documented the process yet. Something I'm working on!
